File: swagger/models/get_contact_logs_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetContactLogsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		contact_logs
		Contains the information about the contact logs.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ContactLogs': 'contact_logs',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'contact_logs': 'ContactLogs',
		}

File: swagger/Models/GetContactLogsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetContactLogsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains the information about the contact logs.
	 * 
	 * @var ContactLog[] 
	 */
	public $contactLogs;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'ContactLogs' => ['contactLogs', 'array', ContactLog::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'contactLogs' => ['ContactLogs', 'array', ContactLog::class],
			];
	}
}
